examples/twittermap/web/public/stylesheets/main.css

```diff
@@ -171,4 +171,55 @@ time-series {
 
 .search-keyword-btn-group{
   margin-left: 36%;
-}
\ No newline at end of file
+}
+
```

Comment:
Also,  could you put some comments about the following changes? Where they are used.